react/tic-tac-toe/src/app/screens/Game/components/Square/index.js

```diff
@@ -1,11 +1,19 @@
-import React from 'react';
-
+import React, { Component } from 'react';
 import styles from './styles.module.scss';
 
-const Square = () => (
-  <button type="button" className={styles.square}>
-    {/* TODO */}
-  </button>
-);
+class Square extends Component {
+
+  render() {
+    const { onClick, value, i } = this.props
+    return (
+      <button
+        className={styles.square}
+        onClick = {() => onClick(i)}
```

Comment:
Dont use arrow functions in onClick because every time the component render create a function reference. 
You can extract the function and then call it from onClick like onClick={this.handleClick}